server/db/models/index.js

```diff
@@ -26,11 +27,18 @@ Review.belongsTo(Product)
 User.hasMany(Review)
 Review.belongsTo(User)
 
+Cart.belongsTo(User)
+Cart.belongsTo(Session)
```

Comment:
Ugh, I need to fix this, will do so.

Reply:
Looks like you are working on this... let me know if I can help.